source/cpu/cpu.go

```diff
@@ -229,7 +229,7 @@ func (s *cpuSource) Discover() error {
 	// Detect RDT features
 	s.features.Flags[RdtFeature] = nfdv1alpha1.NewFlagFeatures(discoverRDT()...)
 
-	// Detect SGX features
+	// Detect available guest protection(SGX,TDX,SEV) features
```

Comment:
nit: SGX is not for guest protection:
```suggestion
	// Detect available trusted execution environment (SGX,TDX,SEV) features
```